src/sdk/object_sdk.js

```diff
@@ -149,6 +149,13 @@ class ObjectSDK {
                     return;
                 }
             }
+
+            dbg.log0('authorize_request_account: allowed_non_fs_buckets: ', ns && ns.write_resource.resource);
+            const is_nsfs_bucket = ns && ns.write_resource.resource.fs_root_path;
```

Comment:
is this how we identify nsfs buckets?

Reply:
in any case I would move this into the validate_allowed_nsfs_account function to encapsulate better

Reply:
yes, I moved it to a function so it'll be clearer.
I moved it inside the function.